tests.lua

```diff
@@ -265,7 +265,7 @@ function LibSerialize:RunTests()
         { -1.5, 6 },
         { -123.45678901235, 10 },
         { -148921291233.23, 10 },
-        { 0/0, 10 },  -- -1.#IND or -nan(ind)
+        { 0/0, 10, nil, 3 },  -- -1.#IND or -nan(ind)
```

Comment:
Is the workaround on lines 208-212 still necessary after this?

Reply:
I suspect it's still required for out-of-game scenarios where the behaviour of NaN is more well-defined and always compares as inequal to other numeric values.